lib/kubeclient/common.rb

```diff
@@ -253,6 +240,29 @@ def proxy_url(kind, name, port, namespace = '')
       end
     end
 
+    def get_pod_log(pod_name, namespace, container: nil, previous: false)
```

Comment:
According to the Kubernetes API docs, "Return previous terminated container logs" if true.

I believe it's for a case where a container terminates in a pod, and is automatically restarted. This is a mechanism to get the logs of the terminated container, rather than the currently running container.